openmsupply/notify
# 186 cold chain alerts should default to enabled

Closes #186

# 👩🏻‍💻 What does this PR do?
 

Default to all alerts turned on for Cold Chain

![image](https://github.com/openmsupply/notify/assets/8287373/43da60fd-df0a-465e-88d6-79606ebb309e)


# 🧪 How has/should this change been tested?

1. Create a new cold chain alert.
1. Check the defaults are correct
1. Check that the existing configurations haven't be inadvertently updated.

## 💌 Any notes for the reviewer?


I've also removed the backwards compatibility code for recipientIds, as I think the only placing which was still using this old method has been updated now.
